bookbotproject/telegrambot.py

```diff
@@ -0,0 +1,51 @@
+import json
+import requests
+import time
+
+TOKEN = "433004356:AAFzeqBEW8_UgEPDOnJ8bnQAPitaR7gLSSo";
+URL = 	"https://api.telegram.org/bot{}/".format(TOKEN);
+
+def send_request(url):
+	res = requests.get(url);
+	content = res.content.decode("utf8");
+	return content;
+
+
+def get_json(url):
+	content = send_request(url);
+	retrieved_json = json.loads(content);
+	return retrieved_json;
+
+
+def get_updates(offset=None):
+	updateURL = URL + "getUpdates?timeout=100"
+	if offset:
+		updateURL += "&offset={}".format(offset);
+	_json = get_json(updateURL);
+	return _json;
+
+def get_last_chat(updates):
+	length = len(updates["result"]);
+	text = updates["result"][length-1]["message"]["text"];
+	chat_id = updates["result"][length-1]["message"]["chat"]["id"];
+	update_id = updates["result"][length-1]["update_id"]; 
+	return [text, chat_id, update_id];
+
+def send_message(chat_id):
```

Comment:
Hi! I think this function can be more general. 
Currently it takes in chat_id, and sends "jamiryo" to that chat.
Can we add a parameter message (making the definition send_message(message, chat_id))? Then we can send that text instead of "jamiryo". (i really like your choice of default response though :3)